ts/util/html-utils.ts

```diff
@@ -0,0 +1,23 @@
+// adds a header cell to a table header
+export function styleColumnHeader(th: HTMLTableCellElement, header: string) {
+  th.className = "p-4 text-white font-semibold bg-grey-300";
+  th.appendChild(document.createTextNode(header));
+}
+
+// adds link-styled text to a div
+export function styleNameLink(div: HTMLDivElement, name: string) {
```

Comment:
Change this to `addLink(container: HTMLElement, text: string, url: string)`
* obvious that it's adding to the DOM
* can be used to append to any element, rather than only a `div`
* fully creates the link
* styling should be on the link (`<a>` tag) itself - not on the container, which may contain other elements in addition to this link
* Hard-code callers to link to "#" for now just for display purposes. We'll update it with the correct URLs later